main.rb

```diff
@@ -0,0 +1,208 @@
+require 'rubygems'
+require 'nokogiri'
+require 'open-uri'
+require 'watir'
+require 'selenium-webdriver'
+require 'json'
+require 'crack'
+require 'headless'
+require 'colorize'
+
+class Accounts
+  attr_accessor :name, :balance, :currency, :nature
+
+  def initialize(name, balance, currency, nature)
+    @name = name.chomp(' >') # оставил потому что это мой ключ для селекта и сравниваю с элементом у которого нет знака >
+    @balance = balance
+    @currency = currency
+    @nature = nature
+  end
+
+  def to_hash
+    {
+    name: @name,
+    balance: @balance,
+    currency: @currency,
+    nature: @nature
+    }
+  end
+end
+
+class Transactions
+  attr_accessor :date, :description, :amount, :currency, :account_name
+
+  def initialize(date, description, amount, currency, account_name)
+    @date = date
+    @description = description
+    @amount = amount
+    @currency = currency
+    @account_name = account_name
+  end
+
+  def to_hash
+    {
+    date: @date,
+    description: @description,
+    amount: @amount,
+    currency: @currency,
+    account_name: @account_name
+    }
+  end
+end
+
+$URL = 'https://my.fibank.bg/oauth2-server/login?client_id=E_BANK'

```

Comment:
You use this constant only in class.  
I recommend you to move this in class and do no use `$`, it will not be required.